docs/custom_promise_types/backup.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+#
+# Backup custom promise type, uses cfengine.sh library located in same dir,
+# and https://github.com/Lex-2008/backup3
```

Comment:
Should the example depend on external resources like that?

Reply:
The other example, `cp.sh` in previous commit, does not rely on anything else, so this is okay. We will move all of these to a more appropriate place soon.